backend/src/Types/DataField.hs

```diff
@@ -122,7 +122,7 @@ data Stronghold
   | MountGundabad
   | Angmar
   | Moria
-  | DolGoldur
+  | DolGuldur
```

Comment:
You can revoke my Tolkien fan-club membership now.

Reply:
💀